eclipse/openvsx
# /api/-/query response size

Fixes #379 

### Testing steps
- Open this PR in Gitpod.
- Let Gitpod initialize. This can take 5 - 10 minutes, because it will download 485 `vscode.bat` extensions and then publish them to the local instance.
- Get `https://8080-<GITPOD_WORKSPACE_ID>.gitpod.io/api/v4/-/query?extensionId=vscode.bat&includeAllVersions=false`. The response should be ~80 KB. The first object in the extensions array is the latest version. All other objects are version links.
- Get `https://8080-<GITPOD_WORKSPACE_ID>.gitpod.io/api/v4/-/query?extensionId=vscode.bat&includeAllVersions=true`. The response should be ~800 KB. The extensions array contains all versions. Only the last object is a version link for the latest version.
- Get `https://8080-<GITPOD_WORKSPACE_ID>.gitpod.io/api/v3/-/query?extensionId=vscode.bat&includeAllVersions=true`. The response should be ~800 KB.
- Get `https://8080-<GITPOD_WORKSPACE_ID>.gitpod.io/api/v2/-/query?extensionId=vscode.bat&includeAllVersions=true`. The response should be ~650 KB.
- For comparison, get `https://8080-<GITPOD_WORKSPACE_ID>.gitpod.io/api/-/query?extensionId=vscode.bat&includeAllVersions=true`. The response should be ~30 MB.
